FRENCH/src/ch01-02-hello-world.md

```diff
@@ -219,10 +219,10 @@ curly bracket on the same line as the function declaration, adding one space in
 between.
 -->
 
-De plus, observez que le corps de la fonction est placée entre des accolades
-`{}`. Rust a besoin de cela partout autour du corps des fonctions. C'est une
+À noter en outre que le corps de la fonction est placé entre des accolades
+`{}`. Rust en a besoin autour du corps de chaque fonction. C'est une
 bonne pratique d'insérer l'accolade ouvrante sur la même ligne que la
-déclaration de la fonction, en ajoutant un espace entre les deux.
+déclaration de la fonction, en ajoutant une espace entre les deux.
```

Comment:
@joeljpresent : are you sure about `une espace` ?

Reply:
I've not seen https://github.com/Jimskapt/rust-book-fr/pull/38/commits/eb72584e49fac533d4d352f29b8fc571693e1ffd#diff-35b027dbafcee636aebc679a42765cf9R118

I was very surprised and checked https://fr.wiktionary.org/wiki/espace#Nom_commun_2, so it seems legit !

Reply:
Yes. *[Espace](https://fr.wikipedia.org/wiki/Espace_(typographie))* is feminine in typography.